packages/lib-classifier/src/store/ClassificationStore.spec.js

```diff
@@ -245,7 +255,7 @@ describe('Model > ClassificationStore', function () {
       // Why is this test here?
       // The observer is in the feedback store
```

Comment:
I think I’ve moved this observer in this PR.

Reply:
Non-blocking comment: Yep. Will these specs move to the `RootStore` spec file? At the very least the comment about the observer can be removed.

Reply:
This should move up to the root store tests. So much is being stubbed out here that I find it hard to follow what's really being tested. When these tests are moved, I'd suggest rewriting them to remove the stubs and test expected feedback state after a classification completes instead.